doc/features/tls/README.md

```diff
@@ -29,9 +31,18 @@ var cluster = Cluster.Builder()
     .Build();
 ```
 
-### Server authentication
+The .NET SSL API will return a `RemoteCertificateNameMismatch` error when the `HostName` does not match the name on the server certificate. By default the driver performs reverse DNS resolution to obtain the `HostName` from the server node's IP address. You can change this by providing a custom hostname resolver in the `SSLOptions.SetHostNameResolver(...)` method:
 
-By default, `Builder.WithSSL()` adds a `RemoteCertificateValidationCallback` that logs any errors returned by .NET's SSL API.
+```csharp
+var cluster = Cluster.Builder()
+    .AddContactPoints(...)
+    .WithSSL(new SSLOptions().SetHostNameResolver(...))
+    .Build();
+```
+
+Alternatively, you can provide your own `RemoteCertificateValidationCallback` to handle the SSL errors and perform your own validation logic. Note that you can introduce security vulnerabilities with this so you should avoid it if you can.
```

Comment:
Is it worth adding an example for this? Or do we consider this as an advanced feature and as such, people should know how to do it?

Reply:
We have an example of a validator for a custom root certificate scenario and I mention it below. Do you think that I should mention it here too?

Reply:
Oh right, there is an inlined example at line 65, then a mention of an external example line 70. That's fine then :-) Maybe just add a `(see below for an example of this)` kind of thing?

Reply:
done!